feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/incoming/AccountIncomingConfigContent.kt

```diff
@@ -194,22 +185,13 @@ internal fun AccountIncomingConfigContent(
                     }
 
                     item {
-                        if (state.imapAutodetectNamespaceEnabled) {
```

Comment:
Is this an intended change?

Reply:
No. Good catch! I think I messed up the rebase.